added params filtered on mobile

cadibemma opened the pull request:
## Summary
- In the vets api repo there is a file that determines what data is automatically filtered from the logs. We should update this to make sure as much of our data is getting through.

## Related issue(s)
- department-of-veterans-affairs/va-mobile-app#6790

## Testing done
- None


## What areas of the site does it impact?
- None

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

kpethtel reviewed (commented):
Can you tell us how you formed this list? I assume you just went through either the controllers or the contracts, but some of them, like os_name and cookie_id don't seem like they would come from those locations. Did you aggregate these from DD logs of params that were filtered out?

Also, why the mix of snake case and camel case? Do we know if the case matters? The existing list only has snake case. 

I think we should probably also add `filter` to the list to accommodate the [filter lib](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Engineering/BackEnd/Architecture/RecordFiltering). It's not in use yet, but we'll want to be ready for it when we do use it.

cadibemma commented:
> Can you tell us how you formed this list? I assume you just went through either the controllers or the contracts, but some of them, like os_name and cookie_id don't seem like they would come from those locations. Did you aggregate these from DD logs of params that were filtered out?
> 
> Also, why the mix of snake case and camel case? Do we know if the case matters? The existing list only has snake case.
> 
> I think we should probably also add `filter` to the list to accommodate the [filter lib](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Engineering/BackEnd/Architecture/RecordFiltering). It's not in use yet, but we'll want to be ready for it when we do use it.

I formed this list by going through the list of DD logs of params and added those that were filtered out that I thought was necessary. I'm not sure if the case matters so I added it just the way it showed up in DD that why the cases are mixed. I also went ahead and added `filter` to the list. Is there any other on the list that you think is necessary/ unnecessary?

aherzberg reviewed (approved):
LGTM.

rmtolmach commented on config/initializers/filter_parameter_logging.rb:
```diff
@@ -11,6 +11,29 @@
   to_date
   qqtotalfilesize
   type
+  folder_id
+  startDate
+  endDate
+  included
+  page
+  useCache
+  number
+  size
+  sort
+  showCompleted
+  excludeProvidedMessage
+  document_id
+  body
+  category
+  subject
+  cookie_id
+  reply_id
+  ids
+  code
+  endpoint_sid
+  message_id
+  os_name
+  filter
```
@cadibemma Would any of these have PII included?

cadibemma replied:
The only one in my opinion with potential to have PII included is 'body' (So I just removed it).

rmtolmach reviewed (approved):
Please monitor logs closely to ensure no PII is present.
